packages/devtools-launchpad/index.js

```diff
@@ -1,9 +1,9 @@
 const developmentServer = require("./bin/development-server");
 const toolboxConfig = require("./webpack.config");
-const tools = require("./src/tools");
+// const tools = require("./src/tools");
```

Comment:
yeah... i should probably drop the async thing... i had to run `node --harmony-async-await` which is silly :)